pkg/cidata/cidata.TEMPLATE.d/boot/11-colorterm-environment.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -eux
+
+! grep -q "COLORTERM" /etc/ssh/sshd_config || exit 0
```

Comment:
I find `cond && action` simpler than `! cond || action`:

```suggestion
grep -q "COLORTERM" /etc/ssh/sshd_config && exit 0
```

But really, why does this need to be so compact? This isn't `perl`, and we don't have to pay extra per line! The action is more important than the condition, so should be at the beginning of a line. The long form still seems the clearest to me:

```suggestion
if grep -q "COLORTERM" /etc/ssh/sshd_config; then
    exit 0
fi
```

Now personally I would be fine with

```shell
exit 0 if grep -q "COLORTERM" /etc/ssh/sshd_config
```

but that isn't valid shell code, and more importantly, many people would disagree with me. 😸

Reply:
I thought it was the error condition, that the script will stop when it "fails" ? But maybe `if` and `&&` are excluded.

Reply:
Your "short form" sounded more ruby than perl ;-)